package.json

```diff
@@ -40,6 +40,7 @@
     "@babel/preset-env": "7.16.0",
     "@babel/preset-typescript": "7.16.7",
     "@jitsi/eslint-config": "4.0.0",
+    "@types/jasmine": "^3.10.3",
```

Comment:
Nit: can you please remove the caret? We use specific versions all across.

Reply:
Change made, and I agree, sensible plan to pin everything